core/src/main/scala/kafka/cluster/Partition.scala

```diff
@@ -948,7 +948,7 @@ class Partition(val topicPartition: TopicPartition,
     // decide whether to only fetch from leader
     val localLog = localLogWithEpochOrException(currentLeaderEpoch, fetchOnlyFromLeader)
 
-    /* Read the LogOffsetMetadata prior to performing the read from the log.
+    /* Read the log offsets prior to performing the read from the log.
```

Comment:
This whole comment seems outdated given the logic in Replica.updateFetchState().

Reply:
Ack. I will try to rephrase. I think what we're trying to say is that the follower might always be considered behind if the log end offset can be advanced between the time of the fetch and the time of the update.